### PR TITLE
feat: add chainId to gql

### PR DESCRIPTION
**Pull request by Kharabet**
https://app.shortcut.com/dexguru/story/30507/fe-multi-chain-explorer-fixes

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_API_URL=<YOUR_PIBLIC_URL>
```
Comment (NemeZZiZZ) [inline .env.sample:1]: i think its better to fill some default data to simply copy-paste this file

File: src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export enum CHAIN_IDS {
+  'CANTO' = 7700,
+}
+export const CHAIN_ID = parseInt(
+  process.env.NEXT_PUBLIC_CHAIN_ID || CHAIN_IDS.CANTO.toString()
```
Comment (NemeZZiZZ) [inline src/constants.ts:5]: ...hmmmm. is it ok?

**Review by NemeZZiZZ — APPROVED** [review]

File: src/constants.ts (hunk 2 above)
Comment (Kharabet) [reply src/constants.ts:5]: I see no better option than fallback to some chain. 

we can simply explode the app in runtime to indicate that env is not configure correctly

Comment (kovalenp) [reply src/constants.ts:5]: yeah we should throw an error if params are not set on app start

Comment (kovalenp) [reply src/constants.ts:5]: there is a smart way: https://dev.to/sukruozdemir/how-to-validate-environment-file-in-nodejs-m2m but it probably an overkill for just two env vars

File: src/components/Page/PageHeader.tsx [hunk 3]
```diff
@@ -6,37 +6,42 @@ import PageMenu from './PageMenu'
...
+    {
+      caption: 'API',
+      href: 'https://developers.dex.guru',
+    },
+    {
+      caption: 'Dex Metrics',
+      href: `https://dex.guru/markets?chain=${chainConfig.shortName}`,
+    },
+  ]
+  chainConfig.features.some((x) => x.name === 'csr') &&
```
Comment (kovalenp) [inline src/components/Page/PageHeader.tsx:26]: Aren't we doing this for each render?

I'd rather pass links as a component prop and resolve it via getStaticProps on page where it's used.

File: src/pages/csr.tsx [hunk 4]
```diff
@@ -115,18 +116,19 @@ const CSRStats: NextPage<CSRStatsProps> = ({ csrStats, chainConfig }) => {
...
-  } = await apolloClient.query({
+  const { data } = await apolloClient.query({
     query: GET_CSR_STATS,
   })
 
   return addApolloState(apolloClient, {
     props: {
-      csrStats,
-      chainConfig,
+      csrStats: data.csrStats || null,
```
Comment (kovalenp) [inline src/pages/csr.tsx:130]:
```suggestion
      csrStats: data?.csrStats,
```

**Review by kovalenp — APPROVED** [review]

**Review by evahteev — APPROVED** [review]

File: src/pages/csr.tsx (hunk 4 above)
Comment (Kharabet) [reply src/pages/csr.tsx:130]: looooong thread why you can't pass undefined ti props in server side next.js 

https://github.com/vercel/next.js/discussions/11209

I tried with nullable operator,  but then got the above error, and changed fallback to `null`

**Pull request merged**